### ISSUE TITLE
Need to have username / password logins for web UI

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/pattoo-web#21]
1. Local database with encrypted passwords generated by a random salt
        2. Using LDAP

**Comment by gill876** [comment]
Would using Flask be suitable to create the login page?
Also, the flask file would manage the database for the username and password.
There would also be an admin page to add more users with different access types.
The web pages would have similar styles to the current Pattoo web pages.

**Comment by palisadoes** [comment]
Yes you’d need to use flask for this task. The real question is what happens after the person logs in. Some suggestions after login.
1. The ability to do all the tasks a person can do from the command line. The underlying libraries are there, the task would be to create the screens to do this.
1. At the moment, the pattoo api will accept any validly formatted data that any agent sends. We need a way to prevent this from automatically happening. One way to do this is to have an authorization screen for newly detected agents. The code to ignore such data and flag it in the console until accepted will be required.
